tests/lib/rules/no-duplicate-imports.js

```diff
@@ -16,73 +16,222 @@ const rule = require("../../../lib/rules/no-duplicate-imports"),
 // Tests
 //------------------------------------------------------------------------------
 
-const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 6, sourceType: "module" } });
+const ruleTester = new RuleTester({
+    parserOptions: { ecmaVersion: 12, sourceType: "module" }
+});
 
 ruleTester.run("no-duplicate-imports", rule, {
     valid: [
-        "import os from \"os\";\nimport fs from \"fs\";",
-        "import { merge } from \"lodash-es\";",
-        "import _, { merge } from \"lodash-es\";",
-        "import * as Foobar from \"async\";",
-        "import \"foo\"",
-        "import os from \"os\";\nexport { something } from \"os\";",
-        {
-            code: "import os from \"os\";\nexport { hello } from \"hello\";",
+        'import os from "os";\nimport fs from "fs";',
```

Comment:
it’s hard to review these test diffs with the unrelated style changes. can they be reverted?

Reply:
I reverted the file, I think now you can see diffs between commits.